src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -241,16 +247,21 @@ private static boolean printStackTraceOnUserExceptions() {
      * Returns the command line program specified, or prints the usage and exits with exit code 1 *
      */
     private static CommandLineProgram extractCommandLineProgram(final String[] args, final List<String> packageList, final List<Class<? extends CommandLineProgram>> classList, final String commandLineName) {
+
         /** Get the set of classes that are our command line programs **/
         final ClassFinder classFinder = new ClassFinder();
         for (final String pkg : packageList) {
+            classFinder.find(pkg, picard.cmdline.CommandLineProgram.class);
             classFinder.find(pkg, CommandLineProgram.class);
         }
         String missingAnnotationClasses = "";
         final Set<Class<?>> toCheck = classFinder.getClasses();
         toCheck.addAll(classList);
```

Comment:
For downstream projects, it may be useful until the CLP interface is included in Barclay (and used in Picard/GATK) to have the possibility to add single Picard classes instead of package-specific...similar to `getClassList`. Maybe the signature of `getClassList` can change in the meanwhile to just return `Class<?>` and check here if it is one of the accepted classes?